apps/settings/js/panels/screen_lock/panel.js

```diff
@@ -9,7 +9,13 @@ define(function(require) {
 
     return SettingsPanel({
       onInit: function(panel) {
-        screenLock.onInit(panel);
+        var elements = {
+          panel: panel,
+          lockscreenEnable: panel.querySelector('.lockscreen-enable'),
+          passcodeEnable: panel.querySelector('.passcode-enable'),
+          passcodeEditButton: panel.querySelector('.passcode-edit')
```

Comment:
Is there any particular reason you move these three lines from ScreenLock to here?